osbs/http.py

```diff
@@ -165,7 +165,9 @@ def iter_lines(self):
             kwargs['chunk_size'] = 1
         try:
             return self.req.iter_lines(**kwargs)
-        except (requests.exceptions.ChunkedEncodingError, httplib.IncompleteRead):
+        except (requests.exceptions.ChunkedEncodingError,
```

Comment:
I don't think so -- decoded_json() is always called on an iterable, and this is a response.iter_lines(). The error handling is in there now.

So in theory decoded_json() doesn't need to handle any of these.